lib/process-services-cloud/src/lib/group/components/group-cloud.component.ts

```diff
@@ -112,9 +111,9 @@ export class GroupCloudComponent implements OnInit, OnChanges {
 
     isDisabled: boolean;
 
-    constructor(private groupService: GroupCloudService) {
-        this.selectedGroupsSubject = new BehaviorSubject<GroupModel[]>(this.selectedGroups);
-        this.searchGroupsSubject = new BehaviorSubject<GroupModel[]>(this.searchGroups);
+    constructor(private identityGroupService: IdentityGroupService) {
+        this.selectedGroupsSubject = new BehaviorSubject<IdentityGroupModel[]>(this.selectedGroups);
```

Comment:
when constructor in invoked the "selectedGroups" is always "[]", so you can initialize that in the declaration:
```
selectedGroups$ = new BehaviorSubject<IdentityGroupModel[]>([]);
```